vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -45,6 +44,7 @@ type EsxVmdkCmd struct {
 const (
 	commBackendName string = "vsocket"
 	maxRetryCount          = 5
+	errBufLen              = 512 // The same err buf length as used in the VMCI client
```

Comment:
nit - is there a way to get it as C.int(C.MAX_BUF_SIZE)  or something like that (maybe change #define there for `const int`)